lib/locales/hu.ts

```diff
@@ -0,0 +1,936 @@
+import { Locale } from '../types/locale';
+
+export const locale: Locale = {
+  id: 'hu',
+
+  seasons: {
+    advent: {
+      season: 'Adventi idő',  // src: https://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=29
+              weekday: 'Advent {{week, romanize}}. hete $t(weekdays:{{dow}})',  // src: https://kattima.hu/hu/app/pray/bences-zsolozsma-l-a-u-d-e-s-advent-ii.-hete-hetfo-80378
```

Comment:
@MMate2007, this one sounds weird to me, however, I could not (yet) find a better source for this string.

Reply:
`Advent I. hete hétfő`.

Explanation:
- `{{week}}` is replaced with the week number in Arabic numerals (e.g. `1`);
- `{{week, romanize}}` returns the week number in Roman numerals (e.g. `IV`);
- `{{dow}}` is replaced with the day of week number (`0` for Sundays, `6` for Saturdays);
- `$t(weekdays:{{dow}})` returns the localised day of week (e.g. `hétfő`):
  - see the `.weekdays` in `hu.ts`;
- `$t(weekdays:{{dow}}, capitalize)` returns the localised day of week with capital initial letter (e.g. `Hétfő`).